### DISCUSSION TITLE
Add support for AVIF

### DISCUSSION THREAD

**Discussion by Wataru-404** [upscayl/upscayl#886] [Ideas]
As title suggest, add support for AVIF?

**Comment by Woisek** [comment]
Convert into i.e. PNG and upscale that? The output will be a different format anyway. 🤷‍♂️

**Comment by NayamAmarshe** [comment]
STB, the library that we're using in Upscayl, only supports reading these formats: `JPG, PNG, TGA, BMP, PSD, GIF, HDR, PIC`

and writing these formats: `BMP, PNG, TGA, HDR, JPG`
